This is because we don't want the qemu-trad used for the emulation to look at
the xenstore area for qdisk.

Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>

diff --git a/hw/xen_machine_fv.c b/hw/xen_machine_fv.c
index 5789a71..6a4979f 100644
--- a/hw/xen_machine_fv.c
+++ b/hw/xen_machine_fv.c
@@ -370,11 +370,11 @@ static void xen_init_fv(ram_addr_t ram_size, int vga_ram_size,
         fprintf(stderr, "%s: xen backend core setup failed\n", __FUNCTION__);
         exit(1);
     }
-    xen_be_register("console", &xen_console_ops);
-    xen_be_register("vkbd", &xen_kbdmouse_ops);
-#ifndef CONFIG_STUBDOM
-    xen_be_register("qdisk", &xen_blkdev_ops);
-#endif
+/*     xen_be_register("console", &xen_console_ops); */
+/*     xen_be_register("vkbd", &xen_kbdmouse_ops); */
+/* #ifndef CONFIG_STUBDOM */
+/*     xen_be_register("qdisk", &xen_blkdev_ops); */
+/* #endif */
 
     pc_machine.init(ram_size, vga_ram_size, boot_device,
 		    kernel_filename, kernel_cmdline, initrd_filename,
